emily/handler/src/api/handlers/chainstate.rs

```diff
@@ -97,28 +97,23 @@ pub async fn get_chainstate_at_height(
     ),
     security(("ApiGatewayKey" = []))
 )]
-#[instrument(skip(context, api_key))]
-pub async fn set_chainstate(
-    context: EmilyContext,
-    api_key: String,
-    body: Chainstate,
-) -> impl warp::reply::Reply {
+#[instrument(skip(context))]
+pub async fn set_chainstate(context: EmilyContext, body: Chainstate) -> impl warp::reply::Reply {
     debug!("Attempting to set chainstate: {body:?}");
     // Internal handler so `?` can be used correctly while still returning a reply.
     async fn handler(
         context: EmilyContext,
-        api_key: String,
         body: Chainstate,
     ) -> Result<impl warp::reply::Reply, Error> {
         // Convert body to the correct type.
         let chainstate: Chainstate = body;
-        let can_reorg = context.settings.trusted_reorg_api_key == api_key;
+        let can_reorg = true;
```

Comment:
QQ: Is the `can_reorg` parameter of `add_chainstate_entry_or_reorg()` still relevant after these changes?

Reply:
At least from what I see this function is called only from handlers from update/set chainstate, and in both of them `can_reorg` is always true now.

However, from naming, `can_reorg` sounds like it can be a wider amount of reasons, so maybe we should keep it. @Jiloc wdyt

Reply:
Nice! [cd2440b](https://github.com/stacks-sbtc/sbtc/pull/1586/commits/cd2440b7cd4373ad914319129481b9d52de6f5dc)